Classes/CharacterDefaults.gd

```diff
@@ -0,0 +1,316 @@
+class_name CharacterDefaults
+const script_name := "character_defaults"
+
+
+
+
+#### starting equipment ########################################################
+#       _             _   _                                _      
+#      | |           | | (_)                              (_)     
+#   ___| |_ __ _ _ __| |_ _ _ __   __ _    ___  __ _ _   _ _ _ __ 
+#  / __| __/ _` | '__| __| | '_ \ / _` |  / _ \/ _` | | | | | '_ \
+#  \__ \ || (_| | |  | |_| | | | | (_| | |  __/ (_| | |_| | | |_) |
+#  |___/\__\__,_|_|   \__|_|_| |_|\__, |  \___|\__, |\__,_|_| .__/
+#                                  __/ |          | |       | |    
+#                                 |___/           |_|       |_|     
+
+const starting_equipment := {
+	
+	
+	
+	
+	knight = [
+		"sword"
+	],
+	
+	
+	
+	
+	battle_mage = [
+		"staff"
+	],
+	
+	
+	
+	
+	berserker = [
+		"axe"
+	],
+	
+	
+	
+	
+	quick_shooter = [
+		"bow", "arrow", "dagger"
+	]
+	
+	
+	
+	
+}
+
+## [flipH, flipV]
+const flip_profile := {
+	"Friendlies/Tex_AnimeAva_01.png": [true, false],
+	"Friendlies/Tex_AnimeAva_17.png": [false, false],
+	"Friendlies/Tex_AnimeAva_28.png": [false, false],
+	"Friendlies/Tex_AnimeAva_51.png": [false, false],
+	"Enemies/MonstersAvatarIcons_61.PNG": [false, false]
+}
+
+
+
+
+#### characters ################################################################
+
+const characters := {
+	
+	
+	
+	
+	#### alrune ################################################################
+	#         _                       
+	#        | |                      
+	#    __ _| |_ __ _   _ _ __   ___ 
+	#   / _` | | '__| | | | '_ \ / _ \
+	#  | (_| | | |  | |_| | | | |  __/
+	#   \__,_|_|_|   \__,_|_| |_|\___|
+	#                                 
+	#                                 
+	
+	alrune = {
+		meta = {
+			name = "Alrune",
+			nickname = "",
+			info = "",
+		},
```

Comment:
This character was meant to test the dynamically created characters, however now it is a good way of testing if companion characters can be loaded in